Document/0x06d-Testing-Data-Storage.md

```diff
@@ -633,26 +639,30 @@ The following is [sample Objective-C code for excluding a file from a backup](ht
 }
 ```
 
-The following is [sample Swift code for excluding a file from a backup](https://developer.apple.com/library/content/qa/qa1719/index.html "How do I prevent files from being backed up to iCloud and iTunes?") on iOS 5.1 and later:
+The following is sample Swift code for excluding a file from a backup on iOS 5.1 and later, see [Swift excluding files from iCloud backup](https://bencoding.com/2017/02/20/swift-excluding-files-from-icloud-backup/) for more information:
 
 ```swift
- func addSkipBackupAttributeToItemAtURL(filePath:String) -> Bool
-    {
-        let URL:NSURL = NSURL.fileURLWithPath(filePath)
-
-        assert(NSFileManager.defaultManager().fileExistsAtPath(filePath), "File \(filePath) doesn't exist")
-
-        var success: Bool
-        do {
-            try URL.setResourceValue(true, forKey:NSURLIsExcludedFromBackupKey)
-            success = true
-        } catch let error as NSError {
-            success = false
-            print("Error excluding \(URL.lastPathComponent) from backup \(error)");
-        }
-
-        return success
-    }
+func excludeFileFromBackup(filePath: URL) -> Bool {
+  var file = filePath
+  var success: Bool
+
+  do {
+      if FileManager.default.fileExists(atPath: file.path) {
+          var res = URLResourceValues()
+          res.isExcludedFromBackup = true
+          try file.setResourceValues(res)
+          success = true
+      } else {
+          success = false
+          print("File \(file) doesn't exist")
+      }
+  } catch {
+      success = false
+      print("Error excluding \(file.lastPathComponent) from backup \(error)")
```

Comment:
maybe we should not print everything everywhere as we try to recommend at various places not to print debug stuff ^^.

Reply:
I converted the debug print messages into Swift 5's `Result` type for this example.